app/routes/cdo.js

```diff
@@ -52,7 +55,16 @@ module.exports = [
   {
     method: 'GET',
     path: '/cdo/{indexNumber}',
-    options: { tags: ['api'] },
+    options: {
+      tags: ['api'],
+      notes: ['Returns the full CDO with dog, person and exemption details'],
```

Comment:
I like the addition of 'notes' on these methods

Reply:
this comes out in swagger docs as well